python/dgl/batched_graph.py

```diff
@@ -305,7 +50,6 @@ def unbatch(graph):
     --------
     batch
     """
-    assert isinstance(graph, BatchedDGLGraph)
```

Comment:
Add a shortcut for `batch_size == 1`?

Reply:
There is an unused `split` function above. Help delete it?

Reply:
DONE.